suites/test_help_documentation.py

```diff
@@ -24,10 +24,13 @@ def help_doc(self):
             utilities.switch_to_latest_tab(self.driver)
             lcc.log_info(self.driver.current_url)
             check_that("page url", self.driver.current_url, contains_string(constants.help_user_guide_url))
-            title_of_user_guide = utilities.find_shadow_dom_element(self.driver,
-                                                                    locators.TITLE_OF_USER_GUIDE_CSS,
-                                                                    locators.USER_GUIDE_PARENT_CSS).text
+            #title_of_user_guide = utilities.find_shadow_dom_element(self.driver,
+            #                                                       locators.TITLE_OF_USER_GUIDE_CSS,
+            #                                                      locators.USER_GUIDE_PARENT_CSS).text
+            title_of_user_guide=utilities.get_text(self.driver,By.CSS_SELECTOR,locators.TITLE_OF_USER_GUIDE_CSS)
             check_that("title of the page", title_of_user_guide, contains_string("Help"))
```

Comment:
Please add check for the word "Guide" displayed below the title
And check that element with id "#rhdocs-content" is displayed